src/js/api/set-playlist.js

```diff
@@ -16,12 +16,4 @@ const setPlaylist = function(model, playlist, feedData = {}) {
     }
 };
 
-export function loadProvidersForPlaylist(model) {
```

Comment:
You should visit this comment (and perhaps the implementation) https://github.com/jwplayer/jwplayer/blob/v8.0.2/src/js/api/setup-steps.js#L42-L52 since you're moving the code it references.

Reply:
Addressed